xpcPlugin/DataManager.cpp

```diff
@@ -668,17 +668,17 @@ namespace XPC
 			return;
 		}
 
-		if (IsDefault(pos[0]))
```

Comment:
Instead of casting these, lets just change the signature of `IsDefault`. It should still work just fine with floats